### PR TITLE
Contributors file for Chapel 1.23

### PR DESCRIPTION
**Pull request by bradcray**
This is the CONTRIBUTORS.md file for the Chapel 1.23 release.

### CODE REVIEW

**Comment by bradcray** [comment]
@ankingcodes @Rapiz1 @rahulghangas @Aniket21mathur @ram-nad @mcdobe100 @immadisairaj @Liannium @parthsarthiprasad @slnguyen @priyank23 @chris980 @xrsh @dgarvit : If you could check to see that I've spelled and alphabetized your names correctly, I'd appreciate it (and gotten your affiliation correct for those who were more than individual contributors).  If there are no problems, give this comment a thumbs-up to indicate that you've checked.  Thanks!

@chapel-lang/core-contributors : If you could make sure I've included all the people whose PRs you've merged for this release, I'd appreciate that.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -4,50 +4,52 @@ Chapel Contributors
...
 * David Longnecker, [HPE]
-* Aniket Mathur, individual contributor
+* Aniket Mathur, [GSoC 2020] student from [Indian Institute of Technology Roorkee]
+* Ben McDonald, [HPE] intern from [Gonzaga University]
+* Erin Melia, individual contributor
+* Ram Nad, individual contributor
+* Sarah Nguyen, [HPE]
 * Nikhil Padmanabhan, [Yale University]
-* Yujia Qiao, individual contributor
+* Parth Prasad, individual contributor
```
Comment (parthsarthiprasad) [inline CONTRIBUTORS.md:33]:
```suggestion
* Parth Sarthi Prasad, individual contributor
```

**Comment by bradcray** [comment]
@CaptainSharf: I just added you due to yesterday's merge.  If you could check that I've spelled and alphabetized your name correctly, I'd appreciate it.

**Comment by CaptainSharf** [comment]
@bradcray , yes my name is spelled correctly

**Pull request merged**